addons/packages/packetfence.spec

```diff
@@ -583,6 +583,10 @@ if [ ! -f /usr/local/pf/conf/ssl/server.crt ]; then
     cat /usr/local/pf/conf/ssl/server.crt /usr/local/pf/conf/ssl/server.key > /usr/local/pf/conf/ssl/server.pem
 fi
 
+# Create OMAPI key
+if [ ! -f /usr/local/pf/conf/pf_omapi_key ]; then
+    echo $(date) $HOSTNAME | base64 > /usr/local/pf/conf/pf_omapi_key
```

Comment:
I would suggest something like that:

/usr/bin/openssl rand -base64 -out /usr/local/pf/conf/pf_omapi_key 32

Much more random, cleaner, faster, zammitier.